source/mainMenu.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2014 Lockheed Martin Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may 
+// not use this file except in compliance with the License. You may obtain 
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, 
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+this.setup = function() {
+    this.scene.gameCam.setCameraTarget( this.target );
+    this.scene.soundManager.playSound( "musicMenuIntro" );
+    this.scene.soundManager.playSound( "musicMenuLoop" );
```

Comment:
There is no main menu scenario with these changes. It doesn't make sense anymore, particularly with the upcoming Mission system.